src/middlewares/jsonBodyParser.js

```diff
@@ -3,8 +3,11 @@ const contentType = require('content-type')
 
 module.exports = () => ({
   before: (handler, next) => {
-    if (handler.event.headers && handler.event.headers['Content-Type']) {
-      const { type } = contentType.parse(handler.event.headers['Content-Type'])
+    const { headers } = handler.event
```

Comment:
Maybe we can add an early return after this line if `headers` is `undefined` and remove any other check below. This should help us to keep the code simple.

Reply:
updated